test/ruby/test_method.rb

```diff
@@ -1136,4 +1136,17 @@ def o.method(m); nil; end
     assert_equal(m, o.:foo)
     assert_nil(o.method(:foo))
   end
+
+  def test_method_reference_freeze_state
+    m = 1.:succ
+    assert(m.frozen?)
```

Comment:
`assert_predicate` will be better for the failure message.

Reply:
@nobu done.

Reply:
and `assert_not_predicate` instead of `assert_equal(..., false)`.

Reply:
@nobu fixed again <3